records/20xx-xx-xx.TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+# TITLE
+
+Date: 20xx-xx-xx
+
+## Status
+
+**DRAFT**.
+
+_VALUES: (DRAFT, IN REVIEW, APPROVED, REJECTED)_
+
+## Context
+
+_TODO: Describe context that triggered the writing of this ADR._
+
+## Options
+
+_TODO: Describe the different options we have._
+
+### Option 1
+
+_TODO: Describe an option._
+
```

Comment:
Maybe have an `## Additional considerations` section here that's optional?  In case there is more info that is useful to understanding the decision.

Reply:
I added that!